pkg/generator/vars.go

```diff
@@ -16,11 +16,13 @@ var replacer *strings.Replacer
 var varPrefix = "var."
 var localPrefix = "local."
 
-var varTemplate = template.Must(template.New("var_file").Parse(`{{range .}}
-variable "{{ . }}" {
-  description = ""
-}
-{{end}}`))
+var varTemplate = template.Must(template.New("var_file").Funcs(template.FuncMap{"sub": sub}).Parse(`{{- $length := len . -}}
+{{- range $i, $v := . -}}
```

Comment:
there is no newline after rendering. 

Could you pleas back description?

Reply:
Added new line the rendering, also fixed the README.md

Reply:
sorry, but Readme and template are missing `description`

Reply:
It was on purpose, I've removed `description` since is not a mandatory field in the same way that `type` and `default` are. I'm using this tool to automatically generate `variables.tf`, when is needed to add `description` and other variables fields, for example when documenting modules, these are added manually